test/scenarios/dependency/molecule/shell/molecule.yml

```diff
@@ -3,7 +3,7 @@ dependency:
   name: shell
   command: >
     bash -c "
-    ansible-galaxy collection install -p '${MOLECULE_EPHEMERAL_DIRECTORY}/collections' community.molecule &&
+    ansible-galaxy collection install -p '${MOLECULE_EPHEMERAL_DIRECTORY}/collections' community.molecule --force &&
```

Comment:
This was added to ensure the collection is installed because the environment is not isolated and the ANSIBLE_COLLECTIONS_PATH not managed when calling with bash.  This will force install the collection even if it exists in a stand location.

The following scenarios will have their collection path set to the ephmeral directory by default.